Game.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <algorithm>
 #include <cstdlib>
+#include <cstring>
```

Comment:
Really needed?

Reply:
Initially I intended to use std::memcpy to copy the tables, but then I switched to a for-loop and I forgot to remove that header. My mistakes.